### PR TITLE
test: sudo to rm btrfs subvol

### PR DESCRIPTION
**Pull request by tchaikov**
"btrfs subvolume {list,delete}" needs root privilege even if the current
user owns this subvol. one can only list/delete he/she is the root, or
if the btrfs volume was mounted with "-o user_subvol_rm_allowed".

Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
@ErwanAliasr1 ping?

**Comment by tchaikov** [comment]
@ErwanAliasr1 ping again?

File: qa/workunits/ceph-helpers.sh [hunk 1]
```diff
@@ -125,15 +125,10 @@ function teardown() {
...
 function __teardown_btrfs() {
     local btrfs_base_dir=$1
-
-    btrfs_dirs=`ls -l $btrfs_base_dir | egrep '^d' | awk '{print $9}'`
-    current_path=`pwd`
-    # extracting the current existing subvolumes
-    for subvolume in $(cd $btrfs_base_dir; btrfs subvolume list . -t |egrep '^[0-9]' | awk '{print $4}' |grep "$btrfs_base_dir/$btrfs_dir"); do
-       # Compute the relative path by removing the local path
-       # Like "erwan/chroot/ceph/src/testdir/test-7202/dev/osd1/snap_439" while we want "testdir/test-7202/dev/osd1/snap_439"
-       local_subvolume=$(echo $subvolume | sed -e "s|.*$current_path/||"g)
```
Comment (ErwanAliasr1) [inline qa/workunits/ceph-helpers.sh]: Why did you remove that part ?

Comment (tchaikov) [inline qa/workunits/ceph-helpers.sh]: @ErwanAliasr1 b/c the absolute path works better for me. and relative path fails as my mount point is "/foo", not the parent directory of testdir.

if this is what you are asking.

Comment (ErwanAliasr1) [inline qa/workunits/ceph-helpers.sh]: Ok, so +1 for it.
